pylintrc

```diff
@@ -94,13 +94,14 @@ confidence=
 # no Warning level messages displayed, use"--disable=all --enable=classes
 # --disable=W"
 disable=
-    no-init,
     fixme,
     locally-disabled,
+    no-init,
     too-few-public-methods,
     too-many-ancestors,
-    unused-argument,
-    protected-access
+    too-many-instance-attributes,
```

Comment:
@smarnach I thought I had commented on this the first time, but I must have forgotten to submit it - can we address pylint's complaint here, or disable it locally if really it doesn't make sense here, rather than disabling it globally? Although that might tie into my comment on https://github.com/open-craft/opencraft/pull/29#discussion_r44755583 and not be necessary once switched back to attributes.